packages/wrangler/src/config/index.ts

```diff
@@ -306,6 +306,17 @@ export function printBindings(bindings: CfWorkerInit["bindings"]) {
 		});
 	}
 
+	if (unsafe !== undefined && unsafe.metadata !== undefined) {
+		output.push({
+			type: "Unsafe Metadata",
+			entries: Object.keys(unsafe.metadata).map((key) => {
+				// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+				// @ts-ignore
+				return { key: key, value: unsafe.metadata[key] as string };
+			}),
+		});
+	}
+
```

Comment:
Same typing problem as was solved below?

Reply:
Because we need to assign here rather than add to an existing list, the best I have been able to come up with is:

> 		let entries: { key: string; value: string }[] = [];
> 		for (const key of Object.keys(unsafe.metadata)) {
> 			entries.push({ key: key, value: unsafe.metadata[key] as string });
> 		}
> 
> 		output.push({
> 			type: "Unsafe Metadata",
> 			entries: entries,
> 		});

Reply:
Regarding the TOML, I'm not really sure what I could do here?  This seems to be generated for us.

Reply:
Yep. Nothing we can do, except cry silently into our pillow.

Reply:
Here is my suggestion for that code block here:

```ts
	if (unsafe?.metadata !== undefined) {
		const entries = Object.entries(unsafe.metadata).map(([key, value]) => ({key, value: `${value}`}));
		output.push({
			type: "Unsafe Metadata",
			entries,
		});
	}
```